devimint/src/version_constants.rs

```diff
@@ -0,0 +1,7 @@
+use lazy_static::lazy_static;
+use semver::Version;
+
+lazy_static! {
+    pub static ref VERSION_0_3_0_ALPHA: Version =
```

Comment:
Since `Version` fields are public, you could just make it a const, and avoid any locks etc. https://docs.rs/semver/latest/semver/struct.Version.html#

Reply:
Oh, actually there might be no const ways to parse the build & pre. Oh well.

Reply:
Yeah, at some point we'll need to parse and handle a result because of the prerelease.